<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCardInfo</class>
 <widget class="QWidget" name="VideoCardInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>115</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>115</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#videoIcon {
	background-color: rgb(0, 0, 0);
	padding: 7px;
	border: none;
	border-radius:12px
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="videoIcon">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="videoCardInfo.qrc">
           <normaloff>:/icons/resources/Icons/video-solid-green.svg</normaloff>:/icons/resources/Icons/video-solid-green.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="sourceLabel">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Source</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dateLabel">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeLabel">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sizeLabel">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="sourceInfo">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Unkown</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dateInfo">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Unkown</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeInfo">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Unkown</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sizeInfo">
           <property name="font">
            <font>
             <family>Inter</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Unkown</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="videoCardInfo.qrc"/>
 </resources>
 <connections/>
</ui>
